classes/condition.php

```diff
@@ -58,15 +59,19 @@ class condition extends \core_availability\condition {
     public function __construct($structure) {
         $manual_modes = ['normal', 'identification'];
 
-        if (!empty($structure->duration)) {
+        if(!empty($structure->duration)) {
```

Comment:
According to the Moodle code style, spacing was already correct.
https://docs.moodle.org/dev/Coding_style#If_.2F_else

Reply:
Fixed in 831a346